webpack.config.js

```diff
@@ -23,6 +24,7 @@ const config = {
         NODE_ENV: JSON.stringify(process.env.NODE_ENV),
       },
     }),
+    new UglifyJsPlugin(),
```

Comment:
why not use Terser? Uglify JS 1 is very old.

Reply:
Because the project uses webpack3 and some old dependencies, which don't work with Terser :(

Reply:
oh I see